openedx/features/content_tagging/rest_api/v1/views.py

```diff
@@ -0,0 +1,153 @@
+"""
+Tagging Org API Views
+"""
+
+from edx_rest_framework_extensions.paginators import DefaultPagination
+from openedx_tagging.core.tagging.rest_api.v1.views import TaxonomyView
+
+
+from ...api import (
+    create_taxonomy,
+    get_taxonomies,
+    get_taxonomies_for_org,
+)
+from .serializers import TaxonomyOrgListQueryParamsSerializer
+from .filters import UserOrgFilterBackend
+
+
+class TaxonomyOrgView(TaxonomyView):
+    """
+    View to list, create, retrieve, update, or delete Taxonomies.
+
+    **List Query Parameters**
+        * enabled (optional) - Filter by enabled status. Valid values: true, false, 1, 0, "true", "false", "1"
```

Comment:
It's worth noting that this `enabled` parameter is only useful for Taxonomy Admins, since everyone else can only see taxonomies where `enabled=True`.